board/stm32fx/stm32fx_config.h

```diff
@@ -49,17 +56,25 @@
 
 #include "drivers/registers.h"
 #include "drivers/interrupts.h"
-#include "drivers/timers.h"
 #include "drivers/gpio.h"
+#include "stm32fx/peripherals.h"
+#include "stm32fx/interrupt_handlers.h"
+#include "drivers/timers.h"
+#include "stm32fx/lladc.h"
+#include "stm32fx/board.h"
+#include "stm32fx/clock.h"
+
+#if (defined(PANDA) && !defined (BOOTSTUB)) || (defined(PEDAL_USB) && !defined (BOOTSTUB))
```

Comment:
```suggestion
#if defined(BOOTSTUB) || (defined(PANDA) && defined (PEDAL_USB))
```

Reply:
#if !defined (BOOTSTUB) && (defined(PANDA) || defined(PEDAL_USB))

Reply:
oops, that's what I meant